### ISSUE TITLE
[Feature request] Untitled Boormarks 

### ISSUE THREAD

**Issue by zouthie** [flackbash/AudioAnchor#117]
Hi. Thank you for the amazing software.

IMO, it would be great if the default bookmark type were "untitled", just marking the time, por maybe a numeral (e.g.: " Untitled bookmark 01")

Thanks for considering it.

Cheers

**Comment by flackbash** [comment]
Yes, I agree. Right now when adding a bookmark without a name, the bookmark is not saved. This is probably not the desired behavior. It would be better to save the bookmark under "Untitled" or "Untitled 01". 

Thanks for the suggestion I will try to implement this soon.

**Comment by zouthie** [comment]
Hey! Thanks YOU for the awesome app!

Cheers

J

Enviado desde ProtonMail móvil

-------- Mensaje original --------

**Comment by flackbash** [comment]
When adding a bookmark without a name, the bookmark is now saved with the title "Untitled". This will be in the next release.

**Comment by zouthie** [comment]
Hey, thanks a lot!! I just tested it and works flawlessly.

Thanks again! Love your work!

Cheers,

J

Enviado desde ProtonMail móvil

-------- Mensaje original --------
